opd_source_table.csv

```diff
@@ -769,6 +769,7 @@ Pennsylvania,Philadelphia,Philadelphia,Philadelphia Police Department,OFFICER-IN
 Pennsylvania,Philadelphia,Philadelphia,Philadelphia Police Department,STOPS,02/26/2014,06/17/2023,07/07/2023,Police investigations of pedestrians and vehicles from the Philadelphia Police Department,https://www.opendataphilly.org/dataset/vehicle-pedestrian-investigations,,phl,MULTI,Carto,datetimeoccur,car_ped_stops,,0.4.1
 Pennsylvania,Pittsburgh,Pittsburgh,Pittsburgh Bureau of Police,ARRESTS,03/11/1998,06/19/2023,07/07/2023,Arrest data contains information on people taken into custody by City of Pittsburgh police officers. More serious crimes such as felony offenses are more likely to result in an arrest. However; arrests can occur as a result of other offenses; such as parole violations or a failure to appear for trial. All data is reported at the block/intersection level with the exception of sex crimes which are reported at the police zone level.,https://data.wprdc.org/dataset/arrest-data,,https://data.wprdc.org/datastore/dump/e03a89dd-134a-4ee8-a2bd-62c40aeebc6f,MULTI,CSV,ARRESTTIME,,,0.2
 Pennsylvania,Pittsburgh,Pittsburgh,Pittsburgh Bureau of Police,PEDESTRIAN CITATIONS,12/10/1992,06/09/2023,07/07/2023,"Non-traffic citations (NTCs also known as ""summary offenses"") document low-level criminal offenses where a law enforcement officer or other authorized official issued a citation in lieu of arrest. These citations normally include a fine.?In Pennsylvania; NTCs often include a notice to appear before a magistrate if the person does not provide a guilty plea. Offenses that normally result in a citation include disorderly conduct; loitering; harassment and retail theft.",https://data.wprdc.org/dataset/non-traffic-citations,,https://data.wprdc.org/datastore/dump/6b11e87d-1216-463d-bbd3-37460e539d86,MULTI,CSV,CITEDTIME,,,
+Pennsylvania,Pittsburgh,Pittsburgh,Pittsburgh Bureau of Police,TRAFFIC STOPS,01/01/2023,03/31/2023,10/29/2023,Incident level traffic stops made by Pittsburgh Police obtained through Pennsylvania Right to Know Act and hosted at https://www.muckrock.com/foi/pittsburgh-130/q1-2023-traffic-stops-144011/,https://cdn.muckrock.com/foia_files/2023/05/10/TrafficStops_RTK04-02-2023_Chamberlain_With_additional_datasets..xlsx,,,2023,Excel,DATE_OF_STOP,,,
```

Comment:
URL is not in proper location. I would check out some of the other rows to better understand where things should go. 

The added data should also include all information that should be filled in. Information for all columns that contain URLs is provided in the issue. There is also a min OPD version column. The min version provides the min OPD version that can read a dataset. In this case, the min version is determined by the fact that this is Excel data. It is often useful to run the tests on the new dataset as it will find some of these issues.